src/modules/bulk-minter/components/change/ChangeDonationField.tsx

```diff
@@ -0,0 +1,295 @@
+        setSearchResults([]);
+        setLoading(false);
+        setIsTyping(false);
+      } else {
+        performSearch(e.target.value);
+      }
+    }, DEBOUNCE_INTERVAL_MS);
+  }, [setIsTyping, setSearchResults, setLoading, performSearch]);
+  
+  return (
+    <>
+      {props.showDonationField && (
+        <Row>
+          <Form.Item
+            name="addDonation"
+            style={{ width: '100%', marginBottom: '0px' }}
+            rules={[
+              { required: true, message: 'Please enter a value' },
+              {
+                message: `You can only add ${MAX_CREATOR_LIMIT} creators`,
+                async validator() {
+                  if (props.creators.length === MAX_CREATOR_LIMIT) {
+                    throw new Error();
+                  }
+                },
+              },
+            ]}
+          >
+            <Input
+              id="charity-input"
+              style={{ margin: '39px 0 -2px', height: 50 }}
+              placeholder="Search for a nonprofit by name or EIN"
+              maxLength={44}
+              required
+              onChange={handleSearch}
+            />
+          </Form.Item>
+          {((searchResults && searchResults.length > 0) || charityLoading || isTyping) && (
+            <Row
+              style={{
+                backgroundColor: '#262626',
+                rowGap: '8px',
+                position: 'relative',
+                padding: '8px',
+                width: '100%',
+                maxHeight: '320px',
+                overflowY: 'scroll',
+              }}
+            >
+              {charityLoading || isTyping ? (
+                <div
+                  style={{
+                    position: 'absolute',
+                    display: 'flex',
+                    justifyContent: 'center',
+                    backgroundColor: 'rgba(0, 0, 0, 0.5)',
+                    alignItems: 'center',
+                    inset: 0,
+                  }}
+                >
+                  <Loading />
+                </div>
+              ) : (
+                ''
+              )}
+
+              {/* dummy div for spinner if no results */}
+              {(charityLoading || isTyping) && searchResults.length === 0 && (
+                <Row
+                  style={{
+                    backgroundColor: '#D9D9D91A',
+                    borderRadius: '3px',
+                    width: '100%',
+                    minHeight: '200px',
+                    margin: '10px 5px 5px 5px',
+                  }}
+                >
+                  &nbsp;
+                </Row>
+              )}
+
+              {searchResults.map((result) => {
+                const { solana_address } = result.crypto;
+                const existingCreators = props.creators.map((c) => c.address);
+                const indexOfDuplicate = existingCreators.findIndex((a) => solana_address === a);
+                const enableSelectButton = indexOfDuplicate === -1;
```

Comment:
Is this actually a duplicate, or just locating a creator that has already been added?

```suggestion
                const creatorAlreadyAdded = existingCreators.findIndex((a) => solana_address === a) !== -1;
                const enableSelectButton = !creatorAlreadyAdded;
```

Reply:
This diff is from a stale commit I believe, can't find that line anymore.

Reply:
Looks like it's still there. It's not a duplicate line, but a poorly named variable. Suggested change is just to use a more digestible pattern